.gitlab-ci.yml

```diff
@@ -27,6 +28,10 @@ before_script:
   - if [[ -f "environment.$CI_COMMIT_REF_NAME" ]]; then
   -   source environment.$CI_COMMIT_REF_NAME
   - fi
+  - export ADMIN_USER_EMAILS=test@ucsc.edu
```

Comment:
See #46 - we should set up a data store email so that we can start populating all of these email fields...

Reply:
@chmreid That's a good idea, I like the idea behind the ticket.  I'll merge this in now so that testing is working on all new branches, and then we can change it later, though this email serves no real purpose currently except to prevent that `KeyError` so it doesn't have to be a real email.